### ISSUE TITLE
Crates are not opened / are sealed off what is in the water where theres some items to find e.g laser cutter, seaway, etc are sealed off but they should not be sealed so its possible to scan it correctly?

### ISSUE THREAD

**Issue by BeholdIsLost** [Papela/Nitrox-Cracked-Mod#26]
### Are you on the latest version of Nitrox Cracked Mod?

Yes, I'm on the latest release of Nitrox Cracked Mod.

### What version of Subnautica are you using?

Mar-2023 71288

### Describe the issue.

The crates, where some of the items are, e.g laser cutter parts, seaway, etc is just sealed off when there is items in, I can scan some of them through the box but alot of the items are stuck inside the box and the crate hasent been opened / is blocked off so its like "closed", singelplayer it works fine but due to I'm playing it with one of my friends

I use epic games version of Subnautica as when it was free once from epic games, I claimed it.

### Describe the steps you took to encounter the bug.

Just starting the MP server and going into the server, even local or someone else joins, its same same bug with crates, that alot of them / all of them are just closed when they should be open where theres items?

**Comment by ISizeITA** [comment]
you can solve the problem of the boxes simply by destroying them with the habitat builder. I don't know why but it makes me delete those boxes via habitat builder leaving the content exposed

**Comment by BeholdIsLost** [comment]
Yes, that did solve the issue, but now the tooth dont seem to drop, scanner room can like locate where the tooth are but when I went to swim with one of my friends, it was nowhere to be seen like it went underground or something with this nitrox as I use subnautica epic games as it was free when I got it so I have to use this has like issues, that they dont drop when they are supposed to drop on the ground so I can pick it up to make emiligated glass if I'm, correct that was the name

**Comment by ITLSupress** [comment]
That sounds like a separate issue I'd recommend closing this one and opening a new one regarding just that problem.
